### DISCUSSION TITLE
WARNING: You should not upload the `.next` directory.

### DISCUSSION THREAD

**Discussion by random-tech** [vercel/community#1782] [Help]
### Summary

When I build my Next.js project using `vercel build` I get this warning (using Vercel CLI 28.2.2). As far as I'm aware I haven't done anything weird to my build-config (other than turning off minification, for unrelated reasons).

Google has been very unhelpful in figuring out how to fix this, does anyone here have any idea what is happening?

**Comment by amyegan** [comment]
Hi @random-tech. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) you can share or is your repo public?
